.github/issue-labler.yml

```diff
@@ -0,0 +1,3 @@
+Status: 
+  Triage:
+    - '(Describe the bug)'
```

Comment:
We probably want more here to cover enhancements, etc.  We can add those as we think of them, though.